src/locale/en.json

```diff
@@ -1188,9 +1188,9 @@
     "LimitDescProfile":"is using lite version of Myriad. You can only send a friend request to users with active wallet.",
     "LimitTitleExperience":"Create Timeline Limit Reached!",
     "LimitTitleFriends":"Unable to Add Friend",
-    "LimitDescExperience":"You are currently using lite version of Myriad. You can only create up to 5 timeline! Connect Web 3.0 Wallet to create more experience",
-    "LimitDescFriends":"Connect Web 3.0 Wallet to send friend request.",
-    "MaybeLater":"Maybe Later",
+    "LimitDescExperiance":"You are currently using lite version of Myriad. You can only create up to 5 experience! Connect Web 3.0 Wallet to create more experience",
```

Comment:
in en.json `LimitDescExperiance`
in id.json `LimitDescExperiance`
in ru.json `LimitDescExperience`

wich one should be use @abdulhakim2902 ?